service/patch.go

```diff
@@ -199,6 +200,16 @@ func (uis *UIServer) schedulePatch(w http.ResponseWriter, r *http.Request) {
 				errors.Wrap(err, "Error finalizing patch"))
 			return
 		}
+
+		if projCtx.Patch.IsGithubPRPatch() {
+			job := units.NewGithubStatusUpdateJobForPatchWithVersion(projCtx.Patch.Version)
```

Comment:
just a heads up that the comment on this function doesn't match the function name